home-manager/home-manager

```diff
@@ -165,7 +165,12 @@ function doEdit() {
 
     setConfigFile
 
-    exec "$EDITOR" "$HOME_MANAGER_CONFIG"
+    # See https://github.com/nix-community/home-manager/issues/1496 as to why
+    # `eval` is preferable to `exec`.
+    # Use `--` to prevent $EDITOR values that start with "-" from being treated
+    # like options to `eval`. See https://unix.stackexchange.com/a/356536/358522
+    # for more information.
```

Comment:
```suggestion
```

I'd argue that, since this practice is so prevalent in the Unix commandline, there is no need to document what the double-dash does.